src/daemon/krillserver.rs

```diff
@@ -237,9 +237,6 @@ impl KrillServer {
 
                     pubserver.create_publisher(req, &system_actor)?;
 
```

Comment:
Why is this removed? (similar to questions in other parts of this PR about why explicit republishing is removed)